### ISSUE TITLE
HTTP requests with Velocypack content to `_api/gharial` fail

### ISSUE THREAD

**Issue by ewoutp** [arangodb/arangodb#2439]
I’m sending a `POST` request to `/_db/mydatabase/_api/gharial` with the following:
- Content-Type: `application/x-velocypack`
- Content-Length: {correct number}
- Accept: `application/x-velocypack`
- Body: binary equivalent of hex 0b2c01446e616d65627570646174655f76657274696365735f7570646174654c656e446966665f7465737403 (which is vpack for `{"name":"update_vertices_updateLenDiff_test"}`)

I’m getting back:
```
{"error":true,"errorNum":400,"errorMessage":"Unexpected token \u000b in JSON at position 0","code":400}
```

when running exactly the same with json encoding (and content-type, accept `application/json`) everything runs fine

testing against this docker image: arangodb/arangodb:3.1.17

**Comment by ewoutp** [comment]
It turns out this issue is fixed in 3.2.devel, but still exists in 3.1.18.

Here's how to reproduce:

- Clone go-driver
- Checkout `velocypack` branch 
- Run:
```
export ARANGODB=arangodb/arangodb:3.1.18
make run-tests-single-vpack-no-auth
```

to see success in 3.2.devel, run:
```
export ARANGODB=arangodb/arangodb-preview:3.2.devel
make run-tests-single-vpack-no-auth
```

**Comment by ewoutp** [comment]
@neunhoef This is what we discussed
